### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for document in "Run on web app for containers dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1399]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish to web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, observe the name property for the document using Inspect

**Actual Result:**
Name property is empty for the document..

**Narrator behaviour:**
Tab

**NVDA behaviour:**
edit  read only
blank

**Expected Result:**
Name property for the document should be "Run on web app for containers".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app for containers-document-inspect](https://user-images.githubusercontent.com/22956130/35684459-ee8505e0-078c-11e8-969e-ce4c7f3250e5.jpg)
![run on web app for containers-document-narrator](https://user-images.githubusercontent.com/22956130/35684465-f1da3bfc-078c-11e8-8a2b-8e6742bec11c.jpg)

**Comment by Eskibear** [comment]
under investigation

**Comment by Eskibear** [comment]
shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.6.0.
Issue still exists
![1399-not fixed](https://user-images.githubusercontent.com/22956130/39797044-aa1e7e7c-5377-11e8-9cda-b759eda14c67.jpg)

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by jdneo** [comment]
Can show the name property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47150805-542a6b80-d30a-11e8-9a21-ff0509c52892.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749838
